client/src/redux/show-cert-saga.js

```diff
@@ -13,7 +12,8 @@ function* getShowCertSaga({ payload: { username, certSlug } }) {
       for (let i = 0; i < messages.length; i++) {
         yield put(createFlashMessage(messages[i]));
       }
-      yield call(navigate, '/');
+      // eslint-disable-next-line no-restricted-globals
+      history.go(-1);
```

Comment:
```suggestion
      yield call(navigate, -1);
```
This is a little more consistent with how we navigate everywhere else in the app.  Also, reach-router should be more a11y friendly than calling the web apis directly.

Reply:
This does not work well for non-signed-in Campers. That is, it causes the flash to not show.